### PR TITLE
unison: add page

### PR DESCRIPTION
**Pull request by wmoreland**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/unison.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# unison
+
+> Bidirectional file synchronisation tool.
+> More information: <https://www.cis.upenn.edu/~bcpierce/unison/download/releases/stable/unison-manual.html>.
+
+- Sync two directories (creates log first time these two directories are synchronised):
+
+`unison root1 root2`
```
Comment (pxgamer) [inline pages/common/unison.md]:
```suggestion
`unison {{path/to/directory_1}} {{path/to/directory_2}}`
```

File: pages/common/unison.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Automatically accept the (non-conflicting) defaults:
+
+`unison root1 root2 -auto`
```
Comment (pxgamer) [inline pages/common/unison.md]:
```suggestion
`unison {{path/to/directory_1}} {{path/to/directory_2}} -auto`
```

File: pages/common/unison.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Ignore some files using a pattern:
+
+`unison root1 root2 -ignore {{pattern}}`
```
Comment (pxgamer) [inline pages/common/unison.md]:
```suggestion
`unison {{path/to/directory_1}} {{path/to/directory_2}} -ignore {{pattern}}`
```

**Comment by wmoreland** [comment]
@pxgamer thank you for the comments, totally agree!

**Comment by wmoreland** [comment]
This was my first time dealing with code review so I had a little trouble understanding how to accept the changes but I think I've got there in the end.

**Comment by wmoreland** [comment]
No, thank you @pxgamer
One issue though: I ran the file through the recommended tdlr linter and it recommended that description lines should end in a full stop, referring to the last line which ends in a URL. I added the fullstop in order to comply with the style guide but I see that this now makes the URL unusable, at least from my termite terminal running the NodeJS client. Shall I open this as a general issue?

**Comment by pxgamer** [comment]
What OS are you using? :slight_smile:

**Comment by wmoreland** [comment]
Arch Linux with 5.3.5 kernel

**Comment by sbrl** [comment]
@wmoreland: Hrm. The URL should be enclosed in `<>`, which is the autolink CommonMark markdown syntax. Your client should render that correctly. Which client are you using?

**Comment by wmoreland** [comment]
@sbrl sorry for the slow reply. I am using the NodeJS client. The URL is enclosed in `<>` as per the style guide and is interpreted as a URL by the NodeJS client but the fullstop which is outside of the closing `>` is included in the URL for some reason.

**Comment by agnivade** [comment]
Right, that is how node client works. It removes the `<>`. Therefore, the `.` gets appended after the URL. I don't see this as a big issue.

**Comment by sbrl** [comment]
It depends if the dot is displayed in the URL link or not. I'd suggest making use of the new url syntax, which works a bit like this:

```bash
URL_START='\e]8;;';
URL_DISPLAY_TEXT='\e\\';
URL_END='\e]8;;\e\\';


# Display a url in the terminal.
# See https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda
# $1 - The url to link to.
# $2 - The display text to show.
display_url() {
	echo -e "${URL_START}$1${URL_DISPLAY_TEXT}$2${URL_END}";
}
```

See https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda

**Pull request merged**